packages/lib-react-components/src/MetaToolsButton/MetaToolsButton.js

```diff
@@ -17,16 +17,21 @@ export const StyledPlainButton = styled(PlainButton)`
 function MetaToolsButton (props) {
   const { disabled, icon, onClick, text } = props
 
-  return (
-    <StyledPlainButton
-      disabled={disabled}
-      icon={icon}
-      labelSize='small'
-      text={text}
-      onClick={onClick}
-      {...props}
-    />
-  )
+  try {
+    return (
+      <StyledPlainButton
+        data-testid="test-meta-tools-button"
+        disabled={disabled}
+        icon={icon}
+        labelSize='small'
+        text={text}
+        onClick={onClick}
+        {...props}
+      />
+    )
+  } catch(err) {
+    console.log('goddamn error', err)
```

Comment:
lol OK maybe this is a debug thing. Don't forget to remove this before this PR is ready to roll.